### DISCUSSION TITLE
About zstash update

### DISCUSSION THREAD

**Discussion by jinboxie** [E3SM-Project/zstash#258] [Questions]
I have a question in terms of using zstash. 
I have zstashed my data on hpss of cori. But then, I would have to do update some data in file and also do some deletes of the files. In this case, I would like to update the data but not update the deleted files. I wonder if the zstash would update the renewed data only, or would also update the deleted data info (by deleting the data also in hpss)?

**Comment by forsyth2** [comment]
There's an example of `zstash update` at https://e3sm-project.github.io/zstash/_build/html/main/tutorial.html#simple-case. Each `zstash update` will create new `tar`s that include any new files, edits to existing files, or deletions. When you `zstash extract` the contents from HPSS, the directory should look as you expect based on the updates you performed.

**Reply by forsyth2** [reply]
(Converted #257 into a discussion post since it is a question and not a feature request.)

**Reply by jinboxie** [reply]
Thank u!
